pkg/bff/api/v1/api.go

```diff
@@ -22,6 +22,7 @@ type BFFClient interface {
 
 	// Authenticated Endpoints
 	AddCollaborator(context.Context, *models.Collaborator) (*models.Collaborator, error)
+	ListCollaborators(context.Context) (*ListCollaboratorsReply, error)
```

Comment:
I'm assuming no pagination is necessary for this?

Reply:
I suppose it depends on the number of collaborators we are expecting organizations to have. Probably not more than a dozen right?

Reply:
Probably not - but perhaps we should set a fixed limit just in case (and for security). Could you create a story to limit the number of collaborators to 50? If a 51st collaborator is added, we should return an error instead.